Packs/GitHubITAdmin/Integrations/GitHub_IT_Admin/GitHub_IT_Admin.py

```diff
@@ -0,0 +1,615 @@
+CUSTOM_MAPPING_UPDATE = demisto.params().get('customMappingUpdateUser')
+
+
+class Client(BaseClient):
+    """
+    Client will implement the service API,
+    and should not contain any Demisto logic.
+    Should only do requests and return data.
+    """
+
+    def __init__(self, base_url, token, org, headers, verify=True, proxy=False):
+        self.base_url = base_url
+        self.token = token
+        self.org = org
+        self.verify = verify
+        self.headers = headers
+        self.session = requests.Session()
+        if not proxy:
+            self.session.trust_env = False
+        self.headers['Authorization'] = 'Bearer ' + self.token
+
+    def http_request(self, method, url_suffix, params=None, data=None, headers=None):
+
+        if not headers:
+            headers = self.headers
+        full_url = self.base_url + url_suffix
+        res = requests.request(
+            method,
+            full_url,
+            verify=self.verify,
+            params=params,
+            json=data,
+            headers=headers
+        )
+
+        return res
+
+    def get_user(self, input_type, user_term):
+
+        if input_type == 'id':
+            uri = f'scim/v2/organizations/{encode_string_results(self.org)}/Users/{encode_string_results(user_term)}'
+
+        else:
+            user_term = "\"" + user_term + "\""
+            uri = f'scim/v2/organizations/{encode_string_results(self.org)}/' \
+                  f'Users?filter={encode_string_results(input_type)} eq {encode_string_results(user_term)}'
+        return self.http_request(
+            method='GET',
+            url_suffix=uri,
+        )
+
+    def create_user(self, data):
+        uri = f'scim/v2/organizations/{encode_string_results(self.org)}/Users'
+        return self.http_request(
+            method='POST',
+            url_suffix=uri,
+            data=data,
+        )
+
+    def update_user(self, user_term, data):
+        uri = f'scim/v2/organizations/{encode_string_results(self.org)}/Users/{encode_string_results(user_term)}'
+        return self.http_request(
+            method='PUT',
+            url_suffix=uri,
+            data=data,
+        )
+
+    def disable_user(self, data):
+
+        uri = f'scim/v2/organizations/{encode_string_results(self.org)}/Users/{encode_string_results(data)}'
+        return self.http_request(
+            method='DELETE',
+            url_suffix=uri
+        )
+
+    # Builds a new user github_user dict with pre-defined keys and custom mapping (for user)
+    def build_github_create_user(self, args, scim):
+        extension_schema = scim.get(SCIM_EXTENSION_SCHEMA)
+
+        github_user = scim
+
+        custom_mapping = None
+        if args.get('customMapping'):
+            custom_mapping = json.loads(args.get('customMapping'))
+        elif CUSTOM_MAPPING_CREATE:
+            custom_mapping = json.loads(CUSTOM_MAPPING_CREATE)
+
+        if custom_mapping and extension_schema:
```

Comment:
Let's go over this in the demo - I believe you are adding additional info to the user here - I'd like to see this in action.

Reply:
Presented in the demo.